src/_data/team.json

```diff
@@ -270,12 +270,12 @@
     "instagram": "instagram.com/kybrdbnd"
   },
   {
-    "name": "Farhaan",
-    "image": "/image/ZadraSol.webp",
+    "name": "Farhaan Bukhsh",
+    "image": "https://avatars.githubusercontent.com/u/7670449?s=400&u=bf6eb65bdf25c98310e3711ca1d8b58a84ae47d6&v=4",
```

Comment:
Add image to `src/image/team` and give that link.

Reply:
@anistark  above images are being loaded from GitHub only :P

Reply:
it'll be on server. site isn't hosted on github

Reply:
Github links will work because those are public URLs.

Reply:
sure. I thought we were downloading each image. Hence suggested.

Just realised it wasn't for team. My bad. Too much around. Got confused.